src/screens/FeedScreen/hooks/useContentFeed.js

```diff
@@ -31,7 +31,7 @@ const useContentFeed = ({navigation}) => {
     if (isShortText) {
       return 'rgba(0, 0, 0, 0.5)';
     }
-    return COLORS.blue;
+    return colors.blue;
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The color constant has been updated from `COLORS.blue` to `colors.blue`. Make sure that `colors.blue` is defined in the `colors` module.
```diff
- return COLORS.blue;
+ return colors.blue;
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->